build/Dockerfile

```diff
@@ -5,12 +5,12 @@ LABEL stage=build
 RUN microdnf -y install rsync make git tar findutils diffutils
 RUN mkdir /build/
 
-# Install go 1.14
+# Install go 1.15
```

Comment:
Please talk to @mansikulkarni96 about the 1.15 image we are using for the release and see if it can be used here.

Reply:
I think it is usable in CI, but an issue that it introduces is it will add a gpg requirement to run the e2e from contributor's dev machines. I think testing with the same builder image we are releasing with is important and we should explore that in https://issues.redhat.com/browse/WINC-477
I added a comment to look into that, if you are fine waiting until that ticket is worked on.

Reply:
We cannot explore using a builder image in [WINC-477](https://issues.redhat.com/browse/WINC-477). I would rather have us use the same compiler across dev, CI and release. 

> an issue that it introduces is it will add a gpg requirement to run the e2e from contributor's dev machines.

Do you mean the gpg pakage? Shouldn't that already be present?